package.json

```diff
@@ -59,6 +62,8 @@
     "s3": "gulp s3",
```

Comment:
is `npm s3` (and `pres3`) what gets run when we deploy the site to staging/production?